tests/features/splash_screen.feature

```diff
@@ -50,3 +49,6 @@ Feature: Splash Screen features
     And I should see the link "Deutsch"
     And I should not see "Български"
     And I should not see "Français"
+    # Clean up after the fact.
+    # @todo Remove this once NEXTEUROPA-5519 is in.
+    And I run drush "vdel" "splash_screen_blacklist --yes"
```

Comment:
Thanks, I didn't realize they were uninstalled, I thought they were merely disabled. Will remove this.